src/Android/Renderers/ExtendedGridRenderer.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Android.Content;
+using Bit.App.Controls;
+using Bit.App.Utilities;
+using Bit.Droid.Renderers;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+
+[assembly: ExportRenderer(typeof(ExtendedGrid), typeof(ExtendedGridRenderer))]
+namespace Bit.Droid.Renderers
+{
+    public class ExtendedGridRenderer : ViewRenderer
+    {
+        private static int? _bgResId;
+
+        public ExtendedGridRenderer(Context context) : base(context) { }
+
+        protected override void OnElementChanged(ElementChangedEventArgs<View> elementChangedEvent)
+        {
+            base.OnElementChanged(elementChangedEvent);
+            if (elementChangedEvent.NewElement != null)
+            {
+                SetBackgroundResource(GetBgResId());
+            }
+        }
+
+        private int GetBgResId()
+        {
+            if (_bgResId == null)
+            {
+                if (ThemeManager.GetTheme(true) == "nord")
```

Comment:
Is there a reason `"nord"` is a special case in the way the renderer handles theming?

Reply:
The other themes fall back on generic gray levels for light vs dark, whereas nord has specific colors unique to it.